### ISSUE TITLE
hope feature added: can use .gitignore like eslint, not just .stylelintignore! what do you think so?

### ISSUE THREAD

**Issue by anlexN** [stylelint/stylelint#2795]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

e.g. "A bug where..."

> Which rule, if any, is this issue related to?

e.g. `number-leading-zero`

> What CSS is needed to reproduce this issue?

e.g.

```css
.foo {
  color: pink;
}
```

> What stylelint configuration is needed to reproduce this issue?

e.g.

```json
{
  "rules": {
    "indentation": ["tab", {
      "except": ["value"]
    }]
  }
}
```

> Which version of stylelint are you using?

e.g. `6.7.1`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

e.g. "CLI with `stylelint --config myconfig *.css`"

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

e.g. "Yes, it's related to SCSS maps."

> What did you expect to happen?

e.g. "No warnings to be flagged."

> What actually happened (e.g. what warnings or errors you are getting)?

e.g. "The following warnings were flagged:"

```shell
test.css
1:21    Expected 2 spaces (indentation)
```

**Comment by evilebottnawi** [comment]
@anlexN your can use `stylelint **/*.css --ignore-path .gitignore`

**Comment by anlexN** [comment]
@evilebottnawi , because i use visual studio code stylelint extention, i hope this feature to be added. It is convienent!
i don't like use stylelint command every time , further more stylelint don't have command the same as --watch.

**Comment by evilebottnawi** [comment]
@anlexN we support this feature not only for command line, seems your should craete issue in `stylelint` extension repo for support this feature, thanks!
